pkg/util/provider/machinecontroller/machine_util.go

```diff
@@ -42,7 +42,6 @@ import (
 	utilstrings "github.com/gardener/machine-controller-manager/pkg/util/strings"
 	utiltime "github.com/gardener/machine-controller-manager/pkg/util/time"
 
-	corev1 "k8s.io/api/core/v1"
```

Comment:
Nice catch. Thanks for the this change. 

Ideally, we keep `corev1` and remove `v1` as v1 could refer to v1 of many different libraries. CoreV1 gives more context.  Anyways should be okay for now.